Amazon.QLDB.Driver.Tests/AsyncQldbDriverTests.cs

```diff
@@ -254,11 +220,8 @@ public async Task TestAsyncExecuteWithFuncLambdaReturnsFuncOutput()
         [TestMethod]
         public async Task TestAsyncExecuteWithFuncLambdaAndRetryPolicyReturnsFuncOutput()
         {
-            var driver = new AsyncQldbDriver("ledgerName", mockClient, 4, NullLogger.Instance);
-
-            driver.Dispose();
-
-            await Assert.ThrowsExceptionAsync<QldbDriverException>(async () => await driver.Execute(async txn =>
+            testDriver.Dispose();
```

Comment:
This seems weird....why is this here?

Reply:
The test is testing that Execute throws an exception after the driver has been disposed.
However the name of the test seems like it is testing something different. I've added a new test that better fits this test name and renamed this test to something more appropriate.